force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -16,13 +16,6 @@
  * 
  * Optional Challenge: Use a trigger handler class to implement the trigger logic.
  */
-trigger ContactTrigger on Contact(before insert) {
-	// When a contact is inserted
-	// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
-
-	//When a contact is inserted
-	// if DummyJSON_Id__c is less than or equal to 100, call the getDummyJSONUserFromId API
-
-	//When a contact is updated
-	// if DummyJSON_Id__c is greater than 100, call the postCreateDummyJSONUser API
+trigger ContactTrigger on Contact(before insert, after insert, after update) {
```

Comment:
Nice, clean trigger implementation where all of the actual logic is delegated elsewhere 😎